lib/seed_helper.rb

```diff
@@ -0,0 +1,52 @@
+# methods which are used in seeds and migrations
+class SeedHelper
```

Comment:
I like extracting it to separate class. I suppose the motivation was for these helpers to be testable but I don't see any SeedHelper test. Or was the motivation different?